rasa/nlu/featurizers/count_vectors_featurizer.py

```diff
@@ -224,17 +224,25 @@ def _get_message_text_by_attribute(
 
         tokens = self._get_message_tokens_by_attribute(message, attribute)
 
-        text = self.process_text(tokens)
+        text = self.process_text(tokens, attribute)
 
         text = self._replace_with_oov_token(text, attribute)
 
         return text
 
-    def process_text(self, tokens: List[Text]) -> Text:
+    def process_text(
```

Comment:
is there a reason why it is public method? I think it should be private
```suggestion
    def _process_text(
```